src/main/java/org/mskcc/cbio/oncokb/domain/UserDetails.java

```diff
@@ -52,6 +52,10 @@ public class UserDetails implements Serializable {
     @JsonIgnoreProperties(value = "userDetails", allowSetters = true)
     private Company company;
 
+    @ManyToOne
+    @JsonIgnoreProperties(value = "serviceUsers", allowSetters = true)
+    private Company serviceCompany;
```

Comment:
In order for the company to be associated with the userDetails table (so I can immediately associate a company with it's service user), I need the one to many relationship from company to user details (the code enforces a 1-1 relationship, but we do not do that in the database in case we want to allow multiple service users.) This also means that we need a many to one relationship in the opposite direction, so I don't get much use out of this field, but it is necessary.